tools/buildPackage.js

```diff
@@ -154,10 +192,29 @@ if (isLinux) {
   cmds.push('copy .\\res\\start-tile-150.png "' + path.join(buildDir, 'resources', 'start-tile-150.png') + '"')
   cmds.push('makensis.exe -DARCH=' + arch + ` res/${channel}/braveDefaults.nsi`)
   cmds.push('ncp ./app/extensions ' + path.join(buildDir, 'resources', 'extensions'))
+
   // Make sure the Brave.exe binary is squirrel aware so we get squirrel events and so that Squirrel doesn't auto create shortcuts.
   cmds.push(`"node_modules/rcedit/bin/rcedit.exe" ./${appName}-win32-` + arch + `/${appName}.exe --set-version-string "SquirrelAwareVersion" "1"`)
 }
 
+// Verify tor binaries and bundle with Brave
+var torPath
+if (isDarwin) {
+  torPath = path.join(buildDir, `${appName}.app`, 'Contents', 'Resources', 'extensions', 'bin')
+} else {
+  torPath = path.join(buildDir, 'resources', 'extensions', 'bin')
+}
+
+cmds.push('mkdirp ' + torPath)
+cmds.push('curl -o ' + path.join(torPath, 'tor') + ' ' + torURL)
+cmds.push('curl -o ' + path.join(torPath, 'tor-sig') + ' ' + torSigURL)
+cmds.push('echo ' + torVerificationPublicKey + ' | gpg --import')
```

Comment:
this throws errors like `/bin/sh: line 3: s2rzCh6Atys212CCZEMs4spWAReXux0KirKq0ON9N8modct2AcNS5bSdmyMhQehP: com
mand not found` on mac since it interprets each line of the public key as a separate command. you can instead write the public key to a file and then do `gpg --import tor_demo_key.asc`